lib/vyasa/sangh/worktable.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Vyasa.Sangh.Workspace do
+  use GenServer
+  # a bit like a tuplespace per disciple
+  # store curr_binding state
+
+  def init(%{topic: topic, disciple: disciple}) do
+    {:ok, %{topic: topic, disciple: disciple, last: %{}}}
+  end
+
+
+  def publish(pid, msg, event) do
+    GenServer.cast(pid, {:publish, msg, event})
+  end
+
+  def last(pid, event) when is_pid(pid) do
+    GenServer.call(pid, {:last, event})
```

Comment:
aligns with the primitive that Procs == users, as mediated by the dynamic supervisor